app/src/main/res/values/strings.xml

```diff
@@ -300,7 +300,8 @@
     <string name="error_invalid_password">The password needs to be at least 6 characters long.</string>
     <string name="user">user</string>
     <string name="error_field_required">This field is required.</string>
-
+    <string name="user_login_url">%s\cgi/user.pl</string>
```

Comment:
We shoud add `translatable="false"`

https://stackoverflow.com/questions/3925072/what-is-the-purpose-of-using-translatable-in-android-strings

Reply:
Okay,I am doing it now